white_label/configure-brand.ts

```diff
@@ -141,6 +138,25 @@ const configureEnv = async (brand: string): Promise<void> => {
   await fs.writeFile('.env', newEnvContent)
 }
 
+const setOrReplaceVariable = (
+  envContent: string,
+  variableName: string,
+  value: string
+): string => {
+  let newEnvContent = envContent
+
+  const userAgentRegex = new RegExp(`^${variableName}=".*"`, 'm')
+  const newUserAgent = `${variableName}="${value}"`
```

Comment:
Variable are still name `userAgentxxx`

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/compare/5c082851d856a57882a9172b53bc26ad61aeaf54..0da0e02cb848ae7014a6c39842bcfbce936597ee